www/base/src/app/builders/builders.controller.js

```diff
@@ -28,6 +28,11 @@ class Builders {
                     active = true;
                 }
             }
+            // virtual builders need to be supplied 'virtual' tag
+            // to consider them as active builders
+            if (builder.tags.includes('virtual')) {
```

Comment:
I would reserve that keyword, using '_virtual_'.
Also, we need to make sure '_virtual_' tag is set for virtual builders
https://github.com/buildbot/buildbot/blob/e8b3696a23335236b69e5ebaa7d7976c5d994f2a/master/buildbot/process/build.py#L271

Reply:
Done